### ISSUE TITLE
Show preview of image attachments

### ISSUE THREAD

**Issue by lkiesow** [opencast/opencast#4342]
In the admin interface, it would be nice if we could render a preview of image attachments (at least JPEG and PNG) at the bottom of the event details dialog:

![Screenshot from 2022-10-19 10-10-06](https://user-images.githubusercontent.com/1008395/196637856-a234e770-14fa-464a-80eb-685b175bc750.png)

We already have the image URL and literally all that's missing is an `<img />` tag.

**Comment by ShivamGarg2910** [comment]
@lkiesow, please assign to me.

**Comment by lkiesow** [comment]
Thanks for taking this on, @ShivamGarg2910

**Comment by Dhananjay-12** [comment]
hi ,is this issue still open ??

**Comment by mtneug** [comment]
@Dhananjay-12, there is #4387, but it seems to be abandoned.

**Comment by lkiesow** [comment]
@Dhananjay-12, if you want to take a shot at this, please do.

**Comment by Dhananjay-12** [comment]
Thanks sir, can you assign it to me??

**Comment by Dhananjay-12** [comment]
Hi, can you help me?
Which file do i have to look for this issue & is there any link associated with this admin interface (I'm a beginner T_T)

**Comment by lkiesow** [comment]
You probably just need to modify https://github.com/opencast/opencast/blob/develop/modules/admin-ui-frontend/app/scripts/shared/partials/modals/event-details.html (maybe a CSS file)

What also may be helpful is that instead of building and running all of Opencast, you can run a development mode of the admin interface with some test data like this:

```
❯ cd dev/opencast
❯ cd modules/admin-ui-frontend 
❯ npm run build
```

**Comment by Dhananjay-12** [comment]
I followed these instruction, 
at last command ```npm run build```  the console throws ```Warning: Error: Cannot find where you keep your Bower packages. Use --force to continue.```

**Comment by Dhananjay-12** [comment]
[Will this work ?]( https://stackoverflow.com/questions/31578794/error-cannot-find-where-you-keep-your-bower-packages-use-force-to-continue )

**Comment by lkiesow** [comment]
Forgot that `bower` is still used here:

```
❯ cd modules/admin-ui-frontend 
❯ npm ci
❯ npx bower install
❯ npm run build
```

This should do the trick.

If you have changes and want a full rebuild, just run the last command again:
```
❯ npm run build
```

**Comment by Dhananjay-12** [comment]
### Has the interface changed? 
   Because  I didn't find the exact dialog box as provided with the issue.
   I have attached the images of dialog box I found.



<img width="627" alt="attachment1" src="https://user-images.githubusercontent.com/86868676/208489470-f8ebb82e-52d4-4b04-a36d-7daecfc8e90f.png">

<img width="607" alt="attachment 2" src="https://user-images.githubusercontent.com/86868676/208489452-c6af7522-7b16-4b9d-833a-9f95b24a4ff0.png">

<img width="653" alt="attachment3" src="https://user-images.githubusercontent.com/86868676/208490253-09a1a477-c5dd-43ab-9678-8d2e5a02c202.png">

**Comment by lkiesow** [comment]
This should hopefully help to explain the task:

https://user-images.githubusercontent.com/1008395/208514834-39d247a5-a010-4998-b2dc-3683ea9e1537.mp4

**Comment by Dhananjay-12** [comment]
Is this good??

<img width="498" alt="result1" src="https://user-images.githubusercontent.com/86868676/208675618-43ff8635-266e-4ba4-a8b3-cac7507223ca.png">

**Comment by lkiesow** [comment]
Yes, something like that was what I was thinking about

**Comment by Dhananjay-12** [comment]
Actually I have to set the height of the div to 2 rem(forced). Like if it goes beyond that the image gets cropped.
And in which CSS/SASS file do I have to make changes??

**Comment by lkiesow** [comment]
>  And in which CSS/SASS file do I have to make changes??

I have no idea. Sorry. Something in the `admin-ui-frontend` directory.

**Comment by mtneug** [comment]
@lkiesow is this fixed by #4556?

**Comment by lkiesow** [comment]
yes
